app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -872,4 +872,17 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
             }
         }
     }
+
+    protected float getPlaybackSpeedForMedia() {
```

Comment:
The controller is about accessing the service and providing fallbacks if it is not started yet. I think this would fit into https://github.com/AntennaPod/AntennaPod/blob/develop/core/src/main/java/de/danoeh/antennapod/core/util/playback/PlaybackController.java#L745. 

Otherwise, we have duplicate code: `MediaplayerActivity.getPlaybackSpeedForMedia` and `PlaybackControlsDialog.getCurrentSpeed`.